pom.xml

```diff
@@ -341,6 +346,11 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.hotels.beans</groupId>
+      <artifactId>bull-bean-transformer</artifactId>
```

Comment:
Is this for testing? I don't think Commons components typically have compile dependencies besides other Commons projects usually.

Reply:
@jvz you can read our long thread about this in the JIRA ticket.  I said the same thing and proposed him some alternatives.

Reply:
Hey @jvz you're  little behind ;-)